src/components/StackedBarChart/index.tsx

```diff
@@ -0,0 +1,369 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import {
+  select,
+  scaleTime,
+  scaleBand,
+  scaleLinear,
+  scaleOrdinal,
+  axisLeft,
+  axisBottom,
+  timeFormat,
+  // timParse,
+  stack,
+} from 'd3';
+import { useStoreState } from '../../state/unistore-hooks';
+
+const dailrain = { '2020-12-30': 10, '2020-12-31': 20 };
+
+const data = [
```

Comment:
Dummy data should be removed

Reply:
removed within commit [30e7678](https://github.com/technologiestiftung/giessdenkiez-de/pull/250/commits/30e7678274417ba302788c934d9294d10b51a3e3)